tests/kaggleSearch.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { expect } from "@playwright/test";
+import { SearchPage } from "../page-object/pages/SearchPage";
+import { test } from "../fixture/fixture";
+
+
+test.describe("Kaggle Search Tests", () => {
+
+  test.beforeEach(async ({ page })=>{
+    await page.goto('https://www.kaggle.com/search');
+  })
+  test("should search for 'bitcoin' and apply various filters", async ({ page }) => {
+    const kagglePage = new SearchPage(page);
+
+    // Search
+    await kagglePage.searchFor('bitcoin')
+    await expect(kagglePage.searchField).toHaveValue('bitcoin');
+
+    // Set Date filter
+    await kagglePage.filterByDate();
+    await expect(kagglePage.filterByDateBtn).toBeVisible();
+
+    // Set Dataset filter
+    await kagglePage.filterByDatasetSize();
+    await expect(kagglePage.filterByDatasetSizeBtn).toBeVisible();
+
+    // Set Dataset filter
+    await kagglePage.filterByFileType();
+    await expect(kagglePage.filterByFileTypeBtn).toBeVisible();
+
+    // // Set License filter
+    // await kagglePage.filterByLicense();
+    // await expect(kagglePage.filterByLicenseBtn).toBeVisible();
+
+    // Перевірити наявність результатів
+    await expect(kagglePage.searchResultItem).toHaveCount(1);
+  });
+  test.only("should display 'No results found' page when no results found", async ({ page }) => {
+    const kagglePage = new SearchPage(page);
+
+    // Search (not existing value)
+    await kagglePage.searchFor('якийсь текст українською');
+    await expect(kagglePage.searchField).toHaveValue('якийсь текст українською');
+
+
+    //Check for content 
+    await expect(kagglePage.noResultsFoundContentTitle).toContainText('No results found for ');
```

Comment:
This test fails all the time for me on 'toContainText' check. It receives an empty string all the time